diff -urNad libdvbcsa-1.0.1-orig/src/dvbcsa_bs_mmx.h libdvbcsa-1.0.1/src/dvbcsa_bs_mmx.h
--- libdvbcsa-1.0.1-orig/src/dvbcsa_bs_mmx.h	2008-11-28 15:14:16.000000000 +0200
+++ libdvbcsa-1.0.1/src/dvbcsa_bs_mmx.h	2010-10-23 11:08:29.000000000 +0300
@@ -49,7 +49,7 @@
 #define BS_SHR(a, n)	_m_psrlqi((a), n)
 #define BS_SHL8(a, n)	BS_SHL(a, 8 * (n))
 #define BS_SHR8(a, n)	BS_SHR(a, 8 * (n))
-#define BS_EXTRACT8(a, n) _mm_cvtsi64_si32(_m_psrlqi((a), 8 * (n)));
+#define BS_EXTRACT8(a, n) _mm_cvtsi64_si32(_m_psrlqi((a), 8 * (n)))
 
 #define BS_EMPTY()	_m_empty()
 
